File: birdmeal/src/main/java/com/backend/birdmeal/dto/UserDto.java
package com.backend.birdmeal.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class UserDto {
    private long userSeq;
    private String userEmail;
    private boolean userRole; // 0이면 일반사용자, 1이면 child
    private String userNickname;
    private String userEoa;
    private String userTel;
    private String userAdd;
    private boolean userIsMint; // 0이면 민팅대상자x, 1이면 민팅대상자
    private int userMonthMoney;
    private boolean userChargeState;
    private String userCreateDate;
    private String userUpdateDate;
}

File: AOS/BirdMeal/app/src/main/java/com/ssafy/birdmeal/model/dto/UserDto.kt
package com.ssafy.birdmeal.model.dto

import com.google.gson.annotations.SerializedName

data class UserDto(
    @SerializedName("userSeq") val userSeq: Int,
    @SerializedName("userEmail") val userEmail: String,
    @SerializedName("userNickname") val userNickname: String,
    @SerializedName("userEoa") val userEoa: String?,
    @SerializedName("userTel") var userTel: String?,
    @SerializedName("userAdd") var userAdd: String?,
    @SerializedName("userChargeState") val userChargeState: Boolean,
    @SerializedName("userRole") val userRole: Boolean, // false: 일반 사용자
    @SerializedName("userIsMint") val userIsMint: Boolean,
    @SerializedName("userMonthMoney") val userMonthMoney: Int
)